src/hotspot/share/runtime/vmThread.cpp

```diff
@@ -324,7 +235,8 @@ void VMThread::wait_for_vm_thread_exit() {
   // Note: it should be OK to use Terminator_lock here. But this is called
   // at a very delicate time (VM shutdown) and we are operating in non- VM
   // thread at Safepoint. It's safer to not share lock with other threads.
-  { MonitorLocker ml(_terminate_lock, Mutex::_no_safepoint_check_flag);
+  {
+    MonitorLocker ml(_terminate_lock, Mutex::_no_safepoint_check_flag);
     while(!VMThread::is_terminated()) {
```

Comment:
Not your bug, but can you add a space before the '('?

Reply:
Fixed